test/TestHelper.cpp

```diff
@@ -132,17 +132,37 @@ void ImportTest::importState(json_spirit::mObject& _o, State& _state)
 
 void ImportTest::importTransaction(json_spirit::mObject& _o)
 {
-	BOOST_REQUIRE(_o.count("nonce")> 0);
-	BOOST_REQUIRE(_o.count("gasPrice") > 0);
-	BOOST_REQUIRE(_o.count("gasLimit") > 0);
-	BOOST_REQUIRE(_o.count("to") > 0);
-	BOOST_REQUIRE(_o.count("value") > 0);
-	BOOST_REQUIRE(_o.count("secretKey") > 0);
-	BOOST_REQUIRE(_o.count("data") > 0);
-
-	m_transaction = _o["to"].get_str().empty() ?
-		Transaction(toInt(_o["value"]), toInt(_o["gasPrice"]), toInt(_o["gasLimit"]), importData(_o), toInt(_o["nonce"]), Secret(_o["secretKey"].get_str())) :
-		Transaction(toInt(_o["value"]), toInt(_o["gasPrice"]), toInt(_o["gasLimit"]), Address(_o["to"].get_str()), importData(_o), toInt(_o["nonce"]), Secret(_o["secretKey"].get_str()));
+    if(_o.count("rlp") > 0)
+    {
+           m_transaction = Transaction(importRlp(_o));
```

Comment:
Why do you need an importRLP function?
The RLP object is not part of a transaction (see YP). The RLP gets created out of the parameters of the transaction.